lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -71,6 +71,85 @@ def test_jumpto_id(self):
         response = self.client.get(jumpto_url)
         self.assertRedirects(response, expected, status_code=302, target_status_code=302)
 
+    def test_jumpto_from_section(self):
+        course = CourseFactory.create()
+        chapter = ItemFactory.create(category='chapter', parent_location=course.location)
+        section = ItemFactory.create(category='sequential', parent_location=chapter.location)
+        expected = 'courses/{course_id}/courseware/{chapter_id}/{section_id}/'.format(
+            course_id=unicode(course.id),
+            chapter_id=chapter.url_name,
+            section_id=section.url_name,
+        )
+        jumpto_url = '{0}/{1}/jump_to/{2}'.format(
+            '/courses',
+            unicode(course.id),
+            unicode(section.location),
+        )
```

Comment:
Yes, it's a good job that they are! Because that's what we're testing for, I'd rather keep them separate in order to make the test more easy to read - inline one can see what is being compared more easily.